src/compiler/commandLineParser.ts

```diff
@@ -1739,14 +1739,23 @@ namespace ts {
         return false;
     }
 
+    export interface TSConfig {

```

Comment:
You want this to be internal since `convertToTSConfig` is internal too.

Reply:
Oke, Thanks.